proxy/http2/HPACK.cc

```diff
@@ -227,7 +256,20 @@ namespace HpackStaticTable
   {
     HpackLookupResult result;
 
-    for (unsigned int index = 1; index < TS_HPACK_STATIC_TABLE_ENTRY_NUM; ++index) {
+    // Limit the search range of static table
+    unsigned int start = 1;
```

Comment:
Maybe comment that the first real header value in TS_HPACK_STATIC_TABLE_ENTRY starts at 1? Without that context, initializing at 1 instead of 0 was surprising to me.